### DISCUSSION TITLE
mogrify-im6.q16: no decode delegate for this image format `' @ error/constitute.c/ReadImage/575.

### DISCUSSION THREAD

**Discussion by jiwencao** [ImageMagick/ImageMagick#6889] [Help]
### ImageMagick version

6.9.11-60 Q16 x86_64

### Operating system, version and so on

WSL2 UBuntu20.04

### Description

My image format is JPEG. When I try to resize these images, the error appears. I'm not sure if this is a bug in WSL2 or if there is a problem with my environment configuration. I hope someone can help. Thanks very much.

**Comment by snibgo** [comment]
Please show the text output from `convert -version`.

What is your ImageMagick command? Please supply a sample input image that shows the problem.

**Reply by jiwencao** [reply]
Thank you for your reply!
I have solved the problem. Thanks anyway.

**Reply by Yunnnnqiiiii** [reply]
Can you tell me how to solve the problem?Thank you!

**Reply by julian-** [reply]
> Thank you for your reply! I have solved the problem. Thanks anyway.

@jiwencao 
Hi, it would be very nice if you could share the solution to us :)

Do we need to update the system imagick library to a later version?

**Comment by snibgo** [comment]
It may help other people if you describe your solution.
